README.adoc

```diff
@@ -56,13 +56,14 @@ Some variables are automatically bound to the Groovy script to allow users to us
 |===
 
 You can interrupt request or response processing by setting the result state to `FAILURE`, providing an HTTP
-status code and a message (this is not mandatory, as the default status is `500 - internal server error`).
+status code and a message (this is not mandatory, as the default status is `500 - internal server error`) and select a response template to apply by setting the key.
 
```

Comment:
```
Request or response processing can be interrupted by setting the result state to `FAILURE`. 
By default, it will throw a `500 - internal server error` but you can override this behavior with the following properties:
 - `code`: An HTTP status code
 - `error`: The error message
 - `key`: The key of a response template
```